### ISSUE TITLE
The prediction converges but resulting in same prediction for all the values of chkData?? And how to predict values using the saved model??

### ISSUE THREAD

**Issue by RKNirmal** [tiagoCuervo/FuzzyTensor#3]
I used your code with the sample data having 2 inputs and 1 output(target value) to predict... training ran and it converges as below.
______________________________________________________________
Log:
Train cost after epoch 0: 256.180145
Train cost after epoch 100: 255.096405
Train cost after epoch 200: 254.066574
Train cost after epoch 300: 253.050385
Train cost after epoch 400: 252.039536
Train cost after epoch 500: 251.031601
Train cost after epoch 600: 250.025452
Train cost after epoch 700: 249.020569
Train cost after epoch 800: 248.016602
Train cost after epoch 900: 247.013275
Elapsed time: 1.634994
Validation loss: 532.910950
Model saved in path: /content/model.ckpt
Elapsed time: 1.634994
Elapsed total script execution time: 7.826581
______________________________________________________________
The output of training is as attached as above.
But after this, when I check with the val_pred, it results in the following same values for all the data...!!!

`[10.303363 10.303363 10.303364 10.303363 10.303363 10.303363 10.303363]`

But the actual value which I gave for the chkLbls is as follows
`[437.4 575.5 407.6 682.  475.3 581.3 646.9]` 

why it is having all values same?? is it correct to have the same value in val_pred after training?

If so, after training, I just saved the tf.session using the following code
`save_path = saver.save(sess, "/content/model.ckpt")
 print("Model saved in path: %s" % save_path)`

After saving the model, how can I use it to predict the model using random values of input?? so that I can check whether the model works fine with varied input??

**Comment by tiagoCuervo** [comment]
Hi! Sorry for the late answer, I've been kinda bussy. Well, first, your training does not seems to be converging at all, and that is probably because you have not normalized your data. My ANFIS implementation expects data to be normalized to have a mean of 0.0 and a standard deviation of 1.0.

To learn how to save and restore trained models please check the [official Tensorflow documentation](https://www.tensorflow.org/tutorials/keras/save_and_load).

I hope it is still helpful. I will be working in this repo again soon, so feel free to ask further questions.
